plotly/tests/test_core/test_jupyter/lib/tape-wrapper.js

```diff
@@ -0,0 +1,22 @@
+var test = require('tape');
+var xhr = require('xhr');
+
+var cnt = 0;
+var noop = function() {};
+
+var post = function(query, data) {
+    var opts = data ? { body: data } : {};
+    xhr.post('/?' + query + '&' + (cnt++), opts, noop);
```

Comment:
I had to keep a counter to _force_ the browsers to make multiple (somewhat identical) requests.

There might be a way of doing this.